[2025-03-19T18:45:17.132+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-19T18:45:17.184+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_Api_v1.get_name scheduled__2024-10-22T00:00:00+00:00 [queued]>
[2025-03-19T18:45:17.208+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_Api_v1.get_name scheduled__2024-10-22T00:00:00+00:00 [queued]>
[2025-03-19T18:45:17.209+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-03-19T18:45:17.242+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2024-10-22 00:00:00+00:00
[2025-03-19T18:45:17.298+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=4105) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-19T18:45:17.301+0000] {standard_task_runner.py:72} INFO - Started process 4122 to run task
[2025-03-19T18:45:17.305+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_Api_v1', 'get_name', 'scheduled__2024-10-22T00:00:00+00:00', '--job-id', '1179', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmpxaygt1oe']
[2025-03-19T18:45:17.318+0000] {standard_task_runner.py:105} INFO - Job 1179: Subtask get_name
[2025-03-19T18:45:17.397+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_Api_v1.get_name scheduled__2024-10-22T00:00:00+00:00 [running]> on host 63fd4b143e0a
[2025-03-19T18:45:17.519+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_Api_v1' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2024-10-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-22T00:00:00+00:00'
[2025-03-19T18:45:17.522+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-19T18:45:17.525+0000] {python.py:240} INFO - Done. Returned value was: Jerry
[2025-03-19T18:45:17.552+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-19T18:45:17.552+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_Api_v1, task_id=get_name, run_id=scheduled__2024-10-22T00:00:00+00:00, execution_date=20241022T000000, start_date=20250319T184517, end_date=20250319T184517
[2025-03-19T18:45:17.614+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-19T18:45:17.630+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-19T18:45:17.631+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
